### ISSUE TITLE
Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

### ISSUE THREAD

**Issue by Sexagesimal-merx** [googlecast/CastAndroidTvReceiver#6]
Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

View this [failed invocation](https://github.com/googlecast/CastAndroidTvReceiver/pull/5/checks?check_run_id=9860298920) of the CLA check for more information.

For the most up to date status, view the checks section at the bottom of the pull request.

_Originally posted by @google-cla[bot] in https://github.com/googlecast/CastAndroidTvReceiver/issues/5#issuecomment-1336024426_

**Comment by joeyparrish** [comment]
Do you need help with the CLA process?

**Comment by gomez-andres** [comment]
Hello, thank you for your contribution, however the samples are being maintained by Google. If there are any suggestions or issues encountered please open another issue.
